packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/index.tsx

```diff
@@ -108,10 +152,18 @@ const useSyncEngineSyncProgress = () => {
         }
       )
     );
+    const disposableBlobSet = currentWorkspace.engine.blob.onBlobSet.on(
+      blobSetEventArgs => {
+        console.log('blob set', blobSetEventArgs);
```

Comment:
use debug logger ?

Reply:
Alright, I'll change to a debug logger. It's currently a draft, so please review again once it's turned into a formal PR.